app/views/container_node/_main.html.haml

```diff
@@ -3,6 +3,7 @@
   .col-md-12.col-lg-6
     = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"), :items => textual_group_properties}
     = render :partial => "shared/summary/textual_multilabel", :locals => {:title => _("Conditions"), :items => textual_group_conditions}
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Labels"), :items => textual_container_labels}
```

Comment:
At first sight I'd say that labels should be close to the properties (move above conditions). Just because labels are part of the "static configuration" (more similar to properties), and conditions are more dynamic (e.g. like a status).

/cc @abonas insights?